### ISSUE TITLE
SEC-432: Acegi Support for OpenID

### ISSUE THREAD

**Issue by spring-projects-issues** [spring-projects/spring-security#693]
["Sami Dalouche":https://jira.spring.io/secure/ViewProfile.jspa?name=samokk](Migrated from ["SEC-432":https://jira.spring.io/browse/SEC-432?redirect=false]) said:

It would be nice to have Acegi support the OpenID (http://en.wikipedia.org/wiki/OpenID) distributed SSO protocol. 
A Java library already exists here : http://www.openidenabled.com/openid/libraries/java/janrain-openid-2006-07-14.tgz/view

So, basically, the remaining work is to integrate that library to Acegi's paradigm.
